webapp/src/Controller/Jury/RejudgingController.php

```diff
@@ -201,6 +187,7 @@ public function indexAction(Request $request)
      */
     public function viewAction(
         Request $request,
+        RejudgingService $rejudgingService,
```

Comment:
Why not use $this->rejudgingService?

Reply:
I probably did this because of the submissionService below. What is the guiding principle here? When to use the function arg vs adding it in the constructor of the controller?

Reply:
I mostly put it in the constructor if > 1 method needs it.

Reply:
Done.